Nodejs/Product/Nodejs/Guids.cs

```diff
@@ -27,14 +27,18 @@ static class Guids
         public const string NodejsDebugLanguageString = "{65791609-BA29-49CF-A214-DBFF8AEC3BC2}";
         public const string NodejsEditorFactoryString = "88941496-93F4-4E37-83AF-AFE087415334";
         public const string NodejsEditorFactoryPromptEncodingString = "C8576E92-EFB6-4414-8F63-C84D474A539E";
-        public const string NodejsLanguageInfoString = "ABD5E8A5-5A35-4BE9-BCAF-E10C1212CB40";
+        //do not remove the curly braces. Without curly braces, the interactive window component will fail to load because
+        //the NodejsLanguageInfoString will be compared to the NodejsLanguageInfoString with curlies and they will not match
+        public const string NodejsLanguageInfoString = "{ABD5E8A5-5A35-4BE9-BCAF-E10C1212CB40}";

```

Comment:
The interactive window issue was a separate issue, actually (MEF). In this case, the issue was that in certain cases language service features (e.g. snippets) would fail to load because some comparisons in native code surround the guid string with curlies, and they'll fail to match unless we also surround the guid string with curlies.